spec/support/test_grpc_web_app.rb

```diff
@@ -19,8 +18,9 @@ def self.build(service_class = TestHelloService)
           resource '*', headers: :any, methods: %i[post options]
         end
       end
+      use Rack::Lint
 
-      run grpc_app
+      run GRPCWeb.rack_app
```

Comment:
This increases integration test coverage to cover the `grpc_web.rb` file as well